Microsoft.Toolkit.Uwp.SampleApp.Shared/Shell.xaml

```diff
@@ -80,8 +80,8 @@
               </animations:Implicit.HideAnimations>
             </win:Border>
 
+            <!-- removed DeferLoadStrategy="Lazy" to work around: https://github.com/unoplatform/Uno.WindowsCommunityToolkit/issues/50 -->
```

Comment:
Looks like this PR is also intertangled with #52?

Reply:
Indeed, I intended to merge this after #52, and avoid having to `git rebase` and wait for the CI.
But, now that #52 is invalid, I will remove it from this history.